addStyles.js

```diff
@@ -171,12 +171,17 @@ function attachTagAttrs(element, attrs) {
 function addStyle(obj, options) {
 	var styleElement, update, remove;
 
+	if (obj.sourceMap && typeof options.sourceRoot === "string") {
```

Comment:
@rockmacaca Something in that direction is what I mean by 'can we avoid adding an extra option'? 🙃 If not possible and the user needs control over this, then it can't be helped 😛